Type.sbvr

```diff
@@ -23,6 +23,9 @@ Term:       JSON
 Term:       Hashed
 	Concept Type: Text
 	Note: A 'Text' type that will automatically be converted to a hash.
+Term:       SHA
```

Comment:
It'd probably be better to call it `SHA256` to differentiate between the different types, since it won't work if the type of SHA is silently changed underneath (and yes, `Hashed` suffers from the same problem unfortunately, albeit to a slightly lesser extent I think)

Reply:
@Page- Yeah I thought to too and tried to make it `SHA256` but I'm afraid our SBVR parser would not allow me to do that, because Identifiers can only contain letters and dashes: https://github.com/resin-io-modules/sbvr-parser/blob/master/sbvr-parser.ometajs#L186-L191 so `SHA256` would lead to a sbvr parser error.